near-sdk/src/near_annotations.rs

```diff
@@ -0,0 +1,25 @@
+//! `#[near]` and `#[near_bindgen]` documentation module
+//!
+//! This is not a real module; here we document the attributes that [`#[near]`](crate::near)
+//! and [`#[near_bindgen]`](crate::near_bindgen) macro use.
+
+/// See [`near_sdk::near #[init]`](macro@crate::near#init-sub-attribute)
+pub fn init() {}
+
+/// See [`near_sdk::near #[payable]`](macro@crate::near#payable-sub-attribute)
+pub fn payable() {}
+
+/// See [`near_sdk::near #[private]`](macro@crate::near#private-sub-attribute)
+pub fn private() {}
+
+/// See [`near_sdk::near #[result_serializer]`](macro@crate::near#result_serializer-sub-attribute)
+pub fn result_serializer() {}
+
+/// See [`near_sdk::near #[handle_result]`](macro@crate::near#handle_result-sub-attribute)
+pub fn handle_result() {}
+
+/// See [`near_sdk::near #[event_json]`](macro@crate::near#event_json-sub-attribute)
```

Comment:
changing to `#[near(event_json(...))]` syntax in link name

Reply:
![image](https://github.com/user-attachments/assets/cfe402e7-5744-4302-9d90-38eaf80fde49)
![image](https://github.com/user-attachments/assets/4e9f0047-b345-4bd1-a42c-5bd5e028b278)

both spots unresolved